.github/workflows/swift.yml

```diff
@@ -21,6 +21,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Set Xcode Version
-        run: sudo xcode-select -s /Applications/Xcode_12.app
+        run: sudo xcode-select -s /Applications/Xcode_13.app
       - name: Build
```

Comment:
You'll need to either need to remove this commit so that it doesn't conflict with the changes from Corey that we just landed, or change this line to 12.4, so that it doesn't overwrite Corey's change.